modules/sql/main.tf

```diff
@@ -0,0 +1,16 @@
+resource "azurerm_mysql_flexible_server" "server" {
+  name                = var.server_name
+  resource_group_name = var.resource_group_name
+  location            = var.location
+  administrator_login = var.admin_username
+  # TODO: password as secret
+  administrator_password = var.admin_password
+}
+
+resource "azurerm_mysql_flexible_database" "database" {
+  resource_group_name = var.resource_group_name
+  server_name         = azurerm_mysql_flexible_server.server.name
+  charset             = "utf8"
+  collation           = "utf8_unicode_ci"
+  name                = var.database_name
+}
```

Comment:
Poti sa parcurgi pasii descrisi la primul raspuns de aici https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files si dupa sa dai save in fiecare fisier iar? Asa cred ca scapam de semnele astea rosii de la final de fisier